.github/workflows/main.yml

```
@@ -0,0 +1,78 @@
+name: CI
+
+on: pull_request
+
+jobs:
+  jjb:
+    runs-on: ubuntu-latest
+    name: Jenkins Job Builder
+    steps:
+      - uses: actions/checkout@v2
+      - name: Setup python
+        uses: actions/setup-python@v1
```

```suggestion
        uses: actions/setup-python@v2
```

can you add a dependabot config like https://github.com/theforeman/obal/blob/master/.github/dependabot.yml so that actions get bumped automatically?